src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length <= 1) {
+    return {};
+  }
+
+  const elements = sourceString.trim().split(';');
+
+  const result = elements.reduce((acc, element) => {
+    const property = element.split(':');
+
+    if (property.length === 2) {
+      const key = property[0].trim();
+      const value = property[1].trim();
+
+      acc[key] = value;
+    }
+
+    return acc;
+  }, {});
```

Comment:
The variable name 'result' could be more descriptive to reflect its purpose, such as 'styleObject'. Remember that variable names should convey meaning about what they represent.